serving/accessing-metrics.md

```diff
@@ -26,9 +28,17 @@ kubectl port-forward --namespace knative-monitoring $(kubectl get pods --namespa
 
 4. Set up an administrator account to modify or add dashboards by signing in with username: `admin` and password: `admin`.  
   * Before you expose the Grafana UI outside the cluster, make sure to change the password.
+  
+## Accessing metrics in Stackdriver
+ You can access metrics in  Stackdriver UI:
+ ```
+ https://app.google.stackdriver.com/metrics-explorer?project=<your stackdriver project id>
```

Comment:
IIUC we don't support this in 0.2, so @samodell @RichieEscarez for guidance.

Reply:
That's right. This is not in 0.2. In 0.2, users won't be able to install Stackdriver config which is in "Stackdriver for both logging and metrics" Section in installing-logging-metrics-traces.md.

Reply:
Today, `master` represents 0.2, so this should not be merged in.

Since we've been heads down working only on 0.2, we have only just cut a [0.1.x docs release](https://github.com/knative/docs/tree/v0.1.x) (effectively tagged those docs as v0.1) and have not created a separate "0.1 development"  branch to allow changes to be merged into the past version.

The question is, how urgent and useful, or needed, is this for 0.1? ie. Do we know that there are users on 0.1 that would benefit from this statement? Or can we wait until the docs are updated for 0.2 before circling back around to make 0.1 fixes?

Could scope this PR to only just 0.2 doc updates and then open an Issue for any updates that need to be pushed back into the 0.1 doc set?

Reply:
The scenario in this pr is in serving repo master branch head. https://github.com/knative/serving/blob/da5b8b770f7abee5c7bc4d29b3833858e2465438/config/config-observability.yaml#L92

If there are devs wanting to try out stackdriver metrics, this pr would be helpful for them. Plus, this pr fixed a few broken commands in installing-logging-metrics-traces.md due to previous refactoring.